### PR TITLE
Add OpenAI as services 

### PR DESCRIPTION
**Pull request by PengHongyiNTU**
Feature Addition: OpenAI's Multimodal Services includes Text2Img, Img2Text, ImgEditing, ImgVariation, Text2Audio, Audio2Text

### CODE REVIEW

File: src/agentscope/service/multi_modality/openai_services.py [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+        )
+    except Exception as e:
+        return ServiceResponse(
+            ServiceExecStatus.ERROR,
+            str(e),
+        )
+
+
+def openai_create_image_variation(
+    image_url: str,
```
Comment (DavdGao) [inline src/agentscope/service/multi_modality/openai_services.py:248]: So here we support both local and web URLs?

**Review by DavdGao — COMMENTED** [review]
lgtm, please update the service function section in tutorial accordingly, and add unit tests for openai services

File: src/agentscope/service/multi_modality/openai_services.py (hunk 1 above)
Comment (PengHongyiNTU) [reply src/agentscope/service/multi_modality/openai_services.py:306]: Yes.  As `agentscope.models.openai_model` does not provide wrappers for editing and creating image variation, I use `_parse_url` function that takes either local or web URLs as input, and return objects `BytesIO` .

**Review by DavdGao — COMMENTED** [review]
LGTM, please update the tutorial file in both Chinese and English

**Review by DavdGao — APPROVED** [review]
lgtm

**Pull request merged**